### PR TITLE
Support for "require('./image.png)" keyword for artwork

### PR DESCRIPTION
**Pull request by jman48**
#### What's this PR do?
It will allow users to use the require keyword to load images for album artwork. e.g 
`MusicControl.setNowPlaying({ artwork: require('../img/album.png') });`
Android only.

#### Which issue(s) is it related to?
https://github.com/tanguyantoine/react-native-music-control/issues/43

#### How to test:
Use the require keyword to load an image and check if it shows as the album artwork

### CODE REVIEW

File: android/src/main/java/com/tanguyantoine/react/MusicControlModule.java [hunk 1]
```diff
@@ -43,6 +46,8 @@
 
     private boolean isPlaying = false;
     private long controls = 0;
+    //Flag to check if artwork is passed using react style "require('./image.png)"
+    private boolean fromRequire = false;
```
Comment (Guichaguri) [inline android/src/main/java/com/tanguyantoine/react/MusicControlModule.java]: Why create another field when you can just pass as a method parameter?

File: index.android.js [hunk 2]
```diff
@@ -22,6 +23,12 @@ var MusicControl = {
     NativeMusicControl.enableBackgroundMode(enable)
   },
   setNowPlaying: function(info){
+    //Check if we have an android asset id from react style image "require('./image.png')"
+    if (info.artwork && !isNaN(info.artwork)) {
+      info.artwork = resolveAssetSource(info.artwork).uri;
```
Comment (Guichaguri) [inline index.android.js]: Wouldn't be simpler if this was handled in the Android side? You just need to check if it's a ReadableMap and get the `uri` from it.

Comment (jman48) [inline index.android.js]: We have to resolve the asset on the JS side as the resolveAssetSource function does not have a java equivalent. If I understand you we should remove the ".uri" and then check on java side if it is a map and extract uri that way?

File: android/src/main/java/com/tanguyantoine/react/MusicControlModule.java (hunk 1 above)
Comment (jman48) [inline android/src/main/java/com/tanguyantoine/react/MusicControlModule.java]: Yeah good call. Will get it changed

File: index.android.js (hunk 2 above)
Comment (Guichaguri) [inline index.android.js]: Yep, that seems like a better solution over `fromRequire`

Comment (jman48) [inline index.android.js]: Latest changes will now detect and extract URI on android side

**Comment by Guichaguri** [comment]
Thank you :)

**Comment by tanguyantoine** [comment]
I would have prefer to wait for iOS implementation but too late :)

**Pull request merged**